webpack.config.js

```diff
@@ -26,15 +33,32 @@ module.exports = {
                 exclude: /node_modules/
             },
             {
-                test: /\.(ts|tsx)$/,
-                loader: 'awesome-typescript-loader',
-                exclude: [/node_modules/]
+                test: /\.(js|jsx)$/,
```

Comment:
Please explain this change...

Reply:
It was a mistake, just changed the typescript loader